index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+
+  <header>
+    <nav>
+      <a href="#"><img src="images/MyLogo.png" alt="Logo"></a>
+      <a href="#"> <img src="images/hamb.png" alt="hamb" class="humb"> </a>
+    </nav>
+  </header>
+  <section class="intro">
+    <div class="head">
+      <h1>
+        I’m Lumato
+        Glad to see you!
+      </h1>
+    </div>
+
+    <section class="content">
+      <p>I’m a software developer! I can help you build a
+        product , feature or website Look through some
+        ce! If you like what you see and have a project you need code
+        d, don’t hestiate to contact me.</p>
+      <p class="connect">LETS CONNECT</p>
+    </section>
+    <section class="icons">
+      <ul>
+        <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="images/in.png" alt="in"></a></li>
+        <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
+        <li><a href="#"><img src="images/cat.png" alt="cat"></a></li>
+        <li><a href="#"><img src="images/fingers.png" alt="fingers"></a></li>
+      </ul>
+    </section>
+
+  </section>
```

Comment:
- [ ] Compare the two images below and fix the following issues in the design:
- The font size,
- The colour,
- Space out the social media links at the bottom,
- The background image size and placement.

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/121543190-94f4b880-c9bd-11eb-8d74-4eb7e4443ab5.png) | ![image](https://user-images.githubusercontent.com/49621279/121543287-a63dc500-c9bd-11eb-9755-d02caf7a61cb.png)